services/src/bin/main.rs

```diff
@@ -6,9 +6,14 @@ use geoengine_services::workflows::registry::HashMapRegistry;
 use geoengine_services::handlers;
 use geoengine_services::users::userdb::HashMapUserDB;
 use geoengine_services::handlers::handle_rejection;
+use geoengine_common::config::CONFIG;
+use std::error::Error;
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
Maybe we should specify our `Result` type here?

Reply:
What's the benefit? This error should never occur in the first place because we can't report it to the user.